forms-flow-api/migrations/versions/0cb5d5bf4c76_new_fields_to_application_audit.py

```diff
@@ -0,0 +1,34 @@
+"""New fields added to application_audit & application
+
+Revision ID: 0cb5d5bf4c76
+Revises: e1ccede6ab26
+Create Date: 2023-07-20 19:28:30.215725
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '0cb5d5bf4c76'
+down_revision = 'e1ccede6ab26'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
```

Comment:
Can you add a script to update existing records for this;
i.e, Update all application with is_resubmit=True, and event_name='application_resubmitted' for all the status we currently allow for resubmit such as `Resubmit`.